mir-ci/mir_ci/tests/display_server_static_file.py

```diff
@@ -0,0 +1,47 @@
+import importlib
+import os
+import tempfile
+from typing import Any
+
+from mir_ci.program.display_server import DisplayServer
+
+
+class DisplayServerStaticFile:
```

Comment:
This can be undone now, not needed for this PR?

Reply:
This file still needs to go.

Reply:
I think I removed it in a previous commit but your force push overwrote that. Should be easy enough to redo :)

Reply:
Nevermind, I actually forgot to remove the file